app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -86,6 +99,23 @@ protected void onCreate(Bundle savedInstanceState) {
 
     }
 
+    public void WiFiDialog(final Activity activity){
```

Comment:
starting method name with capitals? Also should use a verb for function name, like `showXXXX` instead of a noun.